packages/zowe-explorer-ftp-extension/src/extension.ts

```diff
@@ -38,9 +39,10 @@ async function registerFtpApis(): Promise<boolean> {
         zoweExplorerApi.registerMvsApi(new FtpMvsApi());
         zoweExplorerApi.registerJesApi(new FtpJesApi());
 
-        const meta = await CoreUtils.getProfileMeta();
-        await zoweExplorerApi.getExplorerExtenderApi().initForZowe("zftp", meta);
-        await zoweExplorerApi.getExplorerExtenderApi().reloadProfiles("zftp");
+        const schema = await CoreUtils.getProfileSchema();
+        const pType = AbstractFtpApi.getProfileTypeName();
```

Comment:
Not directly related to this PR, but curious why `AbstractFtpApi.getProfileTypeName` is defined twice as both a static method and an instance method in ZoweExplorerAbstractFtpApi.ts?

Reply:
I don't think we need it for the zFTP VSCE since there are no calls to that method and an instantiated FtpApi class. However, it is a required method of the MainframeInteractions.ICommon interface. 😢